omni_robot:

    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 100

    fourth_wheel_joint:
        type: velocity_controllers/JointVelocityController
        joint: fourth_wheel_joint
        p: 0.04
        i: 0.01 
        d: 0.0

    third_wheel_joint:
        type: velocity_controllers/JointVelocityController
        joint: third_wheel_joint
        p: 0.04
        i: 0.01
        d: 0.0

    second_wheel_joint:
        type: velocity_controllers/JointVelocityController
        joint: second_wheel_joint
        p: 0.04 
        i: 0.01 
        d: 0.0
   
    first_wheel_joint:
        type: velocity_controllers/JointVelocityController
        joint: first_wheel_joint
        p: 0.04 
        i: 0.01 
        d: 0.0  
